CMakeLists.txt

```diff
@@ -158,6 +158,11 @@ if(MSVC)
         RUNTIME_OUTPUT_DIRECTORY ${CMAKE_BINARY_DIR}/$<0:>
         LIBRARY_OUTPUT_DIRECTORY ${CMAKE_BINARY_DIR}/$<0:>
         ARCHIVE_OUTPUT_DIRECTORY ${CMAKE_BINARY_DIR}/$<0:>)
+
+    target_compile_options(${PROJECT_NAME} PRIVATE "$<$<CONFIG:Release>:/Zi>")
+    target_link_options(${PROJECT_NAME} PRIVATE "$<$<CONFIG:Release>:/DEBUG>")
```

Comment:
I'm not sue, but we may want /DEBUG:FULL here so that the pdb can be used for debugging outside the build machine as well? We do not have to distribute the pdb, but could make it available.

Reply:
Yeah makes sense.